Tests/test_nodes.py

```
@@ -0,0 +1,169 @@
+    def test_calculate_perfect_accuracy(self):
+        # При совпадающих метках точность должна быть 1.0
+        y_true = np.array([1, 0, 1, 0])
+        y_pred = np.array([1, 0, 1, 0])
+        result = MetricNode.calculate(y_true, y_pred, 'accuracy')
+
+        self.assertAlmostEqual(result[0], 1.0, places=5)
+
+    #  UtilsNode 
+
+    def test_to_json_creates_file(self):
+        # to_json должен создать файл с JSON-конфигурацией модели
```

Ruff (0.15.15)

[warning] Comment contains ambiguous `с` (CYRILLIC SMALL LETTER ES). Did you mean `c` (LATIN SMALL LETTER C)?

(RUF003)